components/wpt-results.html

```diff
@@ -322,6 +322,9 @@
         this.query = ''
         this._refreshDisplayedSpecDirs()
         window.history.pushState({}, '', path)
+        if ('ga' in window) {
```

Comment:
It might help some future readers to add a little line comment here to say that `ga` is Google Analytics.